LibTrinityCore-1.0/LibTrinityCore-1.0.lua

```diff
@@ -39,15 +39,17 @@ end
 
 local frame = CreateFrame("Frame")
 frame:SetScript("OnEvent", function(self, event, prefix, message, channel, sender)
+  PlayerName = table.concat({UnitFullName("player")}, '-') -- "PlayerName-RealmName"
```

Comment:
isn't this just `GetUnitName("player",true)`?

Reply:
also you're polluting the global namespace, don't do that

Reply:
GetUnitName("player",true) just returns the unit name, without the -RealmName part.

Oopsie about the PlayerName thing - fixed